tensorflow/contrib/signal/python/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
i *think* this file can be empty (but check other contrib/../python/ directories)